code/game/jobs/job/supervisor.dm

```diff
@@ -349,3 +351,73 @@
 	bio_chips = list(/obj/item/bio_chip/mindshield)
 	satchel = /obj/item/storage/backpack/satchel_sec
 	dufflebag = /obj/item/storage/backpack/duffel/security
+
+/datum/job/nanotrasentrainer
+	title = "Nanotrasen Career Trainer"
+	flag = JOB_INSTRUCTOR
+	department_flag = JOBCAT_ENGSEC
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the Nanotrasen Representative"
+	department_head = list("Captain")
+	selection_color = "#ddddff"
+	mentor_only = TRUE
+	job_department_flags = DEP_FLAG_COMMAND
```

Comment:
Shouldn't this technically be Legal same as IAA or is it not that important as of now?
```suggestion
	job_department_flags = DEP_FLAG_LEGAL
```

Reply:
gotcha